tests/unit/core/data/test_module.py

```diff
@@ -115,13 +119,17 @@ def test_init(
             train_subset=fxt_config.train_subset,
             val_subset=fxt_config.val_subset,
             test_subset=fxt_config.test_subset,
+            input_size=input_size,
```

Comment:
Maybe we need an additional test to see if the overriding works depending on the position of `input_size`?
Also, could there be a test case to see if it works if `input_size` is None and each transform has a corresponding value?

It would be nice to have a test case to verify that it works in the two cases below.
1. `input_size` is None & `each_subset.input_size` is different.
2. input_size` is None & `each_subset.input_size` is None & `each_transform.scale` is not None.

Reply:
Nice comment. I'll add it for these cases